assets/files/etc/sysconfig/network-scripts/ifcfg-eth1

```diff
@@ -0,0 +1,6 @@
+DEVICE=eth1
+NAME=eth1
```

Comment:
Can you clarify which interface we expect to use here, the expected initial architecture I think is that we have only two, one for external/k8s cluster usage (which can include vlan isolation potentially), and another that's dedicated to provisioning but could potentially be repurposed after initial deployment.

It seems you're planning to repurpose the provisioning nic here?  I was thinking we might reuse it for e.g storage traffic on a dedicated subnet/vlan, but that we probably couldn't use that physical fabric for tenant traffic, as it's potentially going to have a route to BMC IPs etc?

Reply:
no, eth1 is still for external cluster usage, in both physical and virtual env